spec/suites/core/ClassSpec.js

```diff
@@ -35,6 +35,45 @@ describe('Class', () => {
 			expect(method.called).to.be.true;
 		});
 
+		it('calls the correct parent initialize function', () => {
+			const initialize = sinon.spy();
+			const initializeDraw = sinon.spy();
+			const initializeEdit = sinon.spy();
+			const Toolbar = Class.extend({
+				initialize() {
+					initialize();
+				}
+			});
+			const DrawToolbar = Toolbar.extend({
+				initialize(options) {
+					initializeDraw();
+					Toolbar.prototype.initialize.call(this, options);
+				}
+			});
+			const EditToolbar = Toolbar.extend({
+				initialize(options) {
+					initializeEdit();
+					Toolbar.prototype.initialize.call(this, options);
+				}
+			});
+
+			new Toolbar();
+			sinon.assert.calledOnce(initialize);
+			sinon.assert.notCalled(initializeDraw);
+			sinon.assert.notCalled(initializeEdit);
+			initialize.resetHistory();
+
+			new DrawToolbar();
+			sinon.assert.callOrder(initializeDraw, initialize);
+			sinon.assert.notCalled(initializeEdit);
+			initialize.resetHistory();
+			initializeDraw.resetHistory();
+
+			new EditToolbar();
+			sinon.assert.callOrder(initializeEdit, initialize);

```

Comment:
Is this not testing the exact same thing as with `DrawToolbar` above?

Reply:
That makes sure that after calling `Toolbar.prototype.initialize.call(this, options);` in DrawToolbar, `initialize` of EditToolbar is still called.

See #9900

Reply:
I don't see `include()` being called here as mentioned in #9900, so why would the bug occur in this test? Both `DrawToolbar` and `EditToolbar` have the exact same internal logic and the exact same assertions.